### DISCUSSION TITLE
Use fuzz ratio

### DISCUSSION THREAD

**Discussion by mizaki** [comictagger/comictagger#391] [Ideas]
Now there is a fuzz ratio, any objection to a PR to use that to order the series list (and maybe show the value?)?

With the current sort by number of issues and then the option to use by year instead, do either of them make sense as the primary sort? I'm thinking leave the option in but they are now secondary sort options?

**Comment by lordwelch** [comment]
Have you tried the filter publishers option?

**Reply by mizaki** [reply]
I don't follow? Maybe I didn't explain very well:
```
self.ct_search_results = sorted(
                    self.ct_search_results, key=lambda i: str(i["count_of_issues"]), reverse=True
                )
```
Instead of doing that (or year), use the ratio from the fuzz for the series title.

**Reply by lordwelch** [reply]
You can disregard my comment.

That isn't what's used.

see #200 for how results are currently sorted

**Reply by mizaki** [reply]
Looking through https://github.com/comictagger/comictagger/pull/200 obviously the publisher filter can stay as is but everything else can be handled by the fuzz ratio. I'm thinking that the source talker (using the new `ComicVolume`) adds the fuzz ratio to each result then volume window can simply sort by that fuzz ratio value using x as the secondary criteria if optionally chosen.

**Reply by lordwelch** [reply]
This is already calculated in issueidentifier.py look for `id_series_match_identify_thresh` we should just be able to add it to the `IssueResult` dictionary and sort on it

**Reply by mizaki** [reply]
Doesn't look like that will work because ii is only ran if auto-select is used and `search_complete` which does the sort is hit first?

**Comment by lordwelch** [comment]
The current sort in ComicTagger sorts by year and number of issue because most of the time those are most likely the series you are trying to tag, it then prioritizes any issues that exactly or closely match the original search string, which provides a better experience than essentially sorting by series name length
